testing/cpm/CMakeLists.txt

```diff
@@ -59,3 +59,5 @@ add_cmake_config_test( cpm_spdlog-simple.cmake )
 
 add_cmake_config_test( cpm_thrust-export.cmake )
 add_cmake_config_test( cpm_thrust-simple.cmake )
+
+add_cmake_config_test( cpm_cuco.cmake )
```

Comment:
Please rename this to `cpm_cuco-simple.cmake` and add a `cpm_cuco-export` to verify that build and install `EXPORT_SETS` work as expected.